main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
```

Comment:
Does github have anything similar to g4 branch?
Looks like this is branched from server/main.go?
Similar comments to other branched files. 

And we are moving these files one level up? I am not clear the reason behind though.

Reply:
i don't think there is a branch, it can only show diff if it's a "move".

The reason is to have the server folder to hold the server.go as server package. Can not have main.go in there anymore since it has to be a main package as program entrance.